jenkins_jobs/modules/triggers.py

```diff
@@ -511,6 +552,19 @@ def gerrit(registry, xml_parent, data):
         to Gerrit. (default false)
     :arg bool escape-quotes: escape quotes in the values of Gerrit change
         parameters (default true)
+    :arg dict build-cancellation-policy: If used, only running jobs will be
+        cancelled if a new patch version is pushed over.
```

Comment:
I don't get this description. Is it taken from Gerrit Trigger plugin description?

Reply:
Yes, but I will change it to something more readable.